pkg/env/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package env contains helpers for environmental details
+package env
```

Comment:
I needed a better place to put `IsTerminal` and found `pkg/util` which I intend to eventually eliminate (when we overhaul error handling some more, errors are the last thing in that package).

In the future I intend to put more environment detail sniffing in here which will be useful for a few other tasks.

Reply:
(we also moved the GOARCH reading code out of pkg/util here)